### DISCUSSION TITLE
Adding support for cockroachdb?

### DISCUSSION THREAD

**Discussion by Paillat-dev** [shlinkio/shlink#1993] [Ideas]
> CockroachDB is a distributed database with standard SQL for cloud applications. CockroachDB powers companies like Comcast, Lush, and Bose.

I personally like cockroachdb for its ease of use and, easy scalability, and try to use it on most of my projects. It is not very different from postgresql, so that could maybe be officially supported? That's just an idea I am throwing here.

**Comment by acelaya** [comment]
According to this https://github.com/cockroachdb/community-tooling-samples/blob/dc05056660a260be4ea31dcac2c7cf069352cb2b/php/basic-sample.php, looks like there's no need for any extra library, just standard PDO (which is what Shlink ultimately uses) and configure it as if it was a Postgres connection.

I would suggest trying to configure Shlink as if you were going to use Postgres, and see if it works.

Other than that, I cannot promise official support, because it seems like a very niche use case, and the ORM used by Shlink does not officially support it.

**Reply by Paillat-dev** [reply]
Yeah nah that is throwing ssome syntax errors. Tough I am inpressed that it did create some tables. Thank you anyways for your response
